hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllOperation.java

```diff
@@ -72,6 +74,7 @@ public void run() {
         backupEntrySet = new ArrayList<Map.Entry<Data, Data>>();
         int partitionId = getPartitionId();
         final MapServiceContext mapServiceContext = mapService.getMapServiceContext();
+        LocalMapStatsImpl localMapStats = mapServiceContext.getLocalMapStatsProvider().getLocalMapStatsImpl(name);
```

Comment:
This part seems running on remote instead of local. So i guess we can not count it as a local map stat. To add `putAll` count in local map stats, it should be calculated and should be added on caller side or am i missing something?